layout-server/src/main/java/org/finos/vuu/layoutserver/service/LayoutService.java

```diff
@@ -0,0 +1,53 @@
+package org.finos.vuu.layoutserver.service;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.finos.vuu.layoutserver.model.Layout;
+import org.finos.vuu.layoutserver.model.Metadata;
+import org.finos.vuu.layoutserver.repository.LayoutRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Service
+public class LayoutService {
+
+    private final LayoutRepository layoutRepository;
+    private final MetadataService metadataService;
+
+    public Layout getLayout(UUID id) {
+        return layoutRepository.findById(id).orElseThrow();
+    }
+
+    public List<Metadata> getMetadata() {
+        return metadataService.getMetadata();
+    }
+
+    @Transactional
+    public UUID createLayout(Layout layout) {
+        Metadata metadata = metadataService.createMetadata(layout.getMetadata());
+        metadata.setLayout(layout);
+        layout.setMetadata(metadata);
+        return layoutRepository.save(layout).getId();
```

Comment:
discussed offline: change response so we get the whole Layout object rather than the id

Reply:
[ae2d2cc](https://github.com/ScottLogic/finos-vuu/pull/57/commits/ae2d2cc2d1193c7fece1847655a8b22e8c462e09)